hazelcast/include/hazelcast/client/serialization/pimpl/compact.h

```diff
@@ -1522,6 +1529,29 @@ struct HAZELCAST_API field_operations
     static field_kind_based_operations get(enum field_kind field_kind);
 };
 
+namespace rabin_finger_print
```

Comment:
Why don't you do it a class as in Java? We usually used classes instead of namespace at this level.

Reply:
I dıdn't implement it, it was there I just added declarations to the header to expose outside of the translation unit to use in tests.

Reply:
Can we refactor it to a class? Or open an issue to refactor if it will take much time and it is not directly related to your changes?

Reply:
Not a big deal. Refactored.

Fixed